docs/content/en/blog/articles/why-k8ssandra/index.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2021-02-26
+title: "Why K8ssandra?"
+linkTitle: "Why K8ssandra?"
+description: >
+  The inspiration behind the K8ssandra project, and the benefits it brings to developers, operators and SREs.
+author: Jeff Carpenter ([@jscarp](https://twitter.com/jscarp))
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+
+---
+
+## The problem: scalable data persistence for cloud-native applications
+
+The past decade and a half has brought tremendous change in how we in the software industry think about building and delivering internet applications. With the launch of Amazon Web Services (AWS) in 2006, companies of any size could access cloud computing infrastructure. The burst of creativity that followed gave birth to concepts like [cloud-native](http://pzf.fremantle.org/2010/05/cloud-native.html) in 2010, and methodologies such as the [twelve factor app](https://12factor.net/) in 2011.
+
+In parallel, there were major developments in open-source infrastructure for data and computing. [Apache Cassandra](http://cassandra.apache.org/) and other NoSQL databases first appeared around 2008, supporting amazing performance and reliability at internet scale. Infrastructure for deploying and running containerized applications took huge leaps forward with the release of Docker in 2013, and Kubernetes rapidly became the default standard for container orchestration soon after its release in 2016. 
+
+{{< imgproc origins Fit "727x210" >}}
+Computing trends that inspired K8ssandra.
+{{< /imgproc >}}
+
+Unfortunately, computing and data infrastructure have been maturing in largely separate tracks these past several years, driven in part by the initial emphasis on Kubernetes for stateless applications. While many companies have successfully migrated cloud applications to Kubernetes and deployed Cassandra at massive scale in production, the data tier has been slower to follow. Running applications in Kubernetes with databases external to Kubernetes creates a mismatched architecture, which has led to limited developer productivity, duplicative stacks for monitoring applications and database infrastructure, and increased cloud computing cost.
+
+## The solution: K8ssandra == Production-ready Cassandra on Kubernetes
+
+The solution is to move the data tier into Kubernetes. Deploying Cassandra on Kubernetes directly alongside applications can be a significant driver of increased developer productivity and scalability at reduced cost. Whether you are a Cassandra user looking to move clusters to Kubernetes, a Kubernetes user looking for a scalable data solution, or an application developer looking to get up and running quickly with data APIs that “just work”, K8ssandra was created to provide a production-ready deployment of Cassandra on Kubernetes. This includes not only the database itself, but also supporting infrastructure for monitoring and management so that you can deploy with confidence.
```

Comment:
Comma goes inside the quotes:

```suggestion
The solution is to move the data tier into Kubernetes. Deploying Cassandra on Kubernetes directly alongside applications can be a significant driver of increased developer productivity and scalability at reduced cost. Whether you are a Cassandra user looking to move clusters to Kubernetes, a Kubernetes user looking for a scalable data solution, or an application developer looking to get up and running quickly with data APIs that “just work,” K8ssandra was created to provide a production-ready deployment of Cassandra on Kubernetes. This includes not only the database itself, but also supporting infrastructure for monitoring and management so that you can deploy with confidence.
```

Reply:
Do native curly quotes work right in all browsers? I'm not sure if Markdown handles that for us.